extension/goplugin/http.go

```diff
@@ -57,6 +57,9 @@ func (http *HTTP) RequestRaw(ctx context.Context, method, rawURL string, headers
 		resp, err = net_http.DefaultTransport.RoundTrip(req)
 		close(done)
 	}()
+
+	<-done
```

Comment:
this whole approach with spawning goroutine is pretty pointless. Why not simply call:

```
resp, err = net_http.DefaultTransport.RoundTrip(req)
```
?

Reply:
Agree, I'll change that